include/io.tcc

```diff
@@ -0,0 +1,42 @@
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <sstream>
+#include <vector>
+
+#include "io.h"
+
+#include "json.hpp"
+
+//! \brief Write output file for point
+//! \details Get vectors from gmsh.h and output to file here
+void write_output_vertices(
```

Comment:
This is defined as a global function, how does it know it is for `IO` class member function?